configuration/forms.md

```diff
@@ -344,6 +344,17 @@ Forms can include arbitrary binary data which is submitted and included in the d
 
 To mark an element as having binary data add an extra column in the XLSForm called `instance::type` and specify `binary` in the element's row.
 
+## Custom XForm Widgets
+
+Some XForm widgets have been added or modified for use in the app:
+- **Bikram Sambat Datepicker**: Calendar widget using Bikram Sambat calendar. Used by default for appropriate languages.
```

Comment:
Is there any special appearance for this?

Reply:
I couldn't find out any more info about it. @mukesh2006 have you used this before? Do you need to set anything in particular in the form to make it show?

Reply:
@abbyad Yes, we've used **Bikram Sambat Datepicker** in [MPDSR form](https://drive.google.com/open?id=1uiGWJU0178msdiFyb5DwEYQwJvZz6TEM) in [dpho-banke](https://dpho-banke.app.medicmobile.org/). 

We don't have to set anything in the form to show the **Nepali Datepicker**, rather the same form shows **Anno Domini datepicker** with standard medic collect app and shows **Bikram Sambat Datepicker** with this [specific collect app](https://drive.google.com/open?id=0B0QmfLAoUDuITEc2MTVYUzVldjA). i.e. datepicker depends on which collect app is being used rather than any settings in the form.

I noticed few glitches in the Nepali Datepicker:
1. It accepts 99 as a valid date (shown in the screenshot below); Nepali dates can go upto 32, not more than that.
2. There is no validation for year. It accepts 0 as a valid year.  

![image](https://user-images.githubusercontent.com/4347171/54974818-639a9880-4fbd-11e9-916b-2cf5791da94f.png)

Jotting down few desired features: 
1. Validation for date and year. 
3. Choice for user to switch between **Bikram Sambat** and **Anno Domini** in the datepicker. Maybe a toggle button on BS vs AD selection.
4. It will be convenient if the **Bikram Sambat Datepicker** also showed the **Anno Domini date** as subscript. e.g. https://www.hamropatro.com/  
![image](https://user-images.githubusercontent.com/4347171/54973247-30550b00-4fb7-11e9-9d1a-cc26a76f97da.png)